pyproject.toml

```diff
@@ -1,3 +1,70 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0"]
+
+[project]
+name = "pymc-marketing"
+description = "Marketing Statistical Models in PyMC"
+requires-python = ">=3.8"
+readme = "README.md"
+license = {file = "LICENSE"}
+dynamic = ["version"]
+maintainers = [{ name = "PyMC Labs", email = "info@pymc-labs.io" }]
+
+dependencies = [
+    "arviz>=0.13.0",
+    "matplotlib==3.5.1",
+    "numpy>=1.17,!=1.24.0",
+    "pandas",
+    "pymc>=5.0.0",
+    "scikit-learn>=1.1.1",
+    "seaborn>=0.12.1",
+    "xarray"
+]
+
+[project.optional-dependencies]
+docs = [
+    "sphinx",
+    "ipython!=8.7.0",
+    "myst-parser",
+    "myst-nb",
+    "pydata-sphinx-theme>=0.12.0.dev0",
+    "sphinx-copybutton",
+    "sphinx-autodoc-typehints",
+    "sphinxext-opengraph",
+    "sphinx-notfound-page",
+    "sphinx-design"
+]
+lint = [
+    "black>=22.3.0",
+    "flake8>=4.0.1",
+    "isort>=5.10.1",
+    "pre-commit>=2.19.0"
+]
+test = [
+    "lifetimes==0.11.3",
+    "pytest==7.0.1",
+    "pytest-cov==3.0.0"
+]
+
+[tool.setuptools]
+packages = [
+    "pymc_marketing", 
+    "pymc_marketing.mmm", 
+    "pymc_marketing.clv", 
+    "pymc_marketing.clv.models"
```

Comment:
@tomicapretto why was this line `clv.models` needed?

Reply:
So it's possible to do `from pymc_marketing.clv.models import ...`. 

See this build that failed https://readthedocs.org/projects/pymc-marketing/builds/19423628/

```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/pymc-marketing/envs/146/lib/python3.10/site-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pymc-marketing/checkouts/146/docs/source/conf.py", line 5, in <module>
    import pymc_marketing  # isort:skip
  File "/home/docs/checkouts/readthedocs.org/user_builds/pymc-marketing/envs/146/lib/python3.10/site-packages/pymc_marketing/__init__.py", line 1, in <module>
    from pymc_marketing import clv, mmm
  File "/home/docs/checkouts/readthedocs.org/user_builds/pymc-marketing/envs/146/lib/python3.10/site-packages/pymc_marketing/clv/__init__.py", line 1, in <module>
    from pymc_marketing.clv.models import (
ModuleNotFoundError: No module named 'pymc_marketing.clv.models'
```